source/fundamentals/enterprise-auth.txt

```diff
@@ -1,27 +1,186 @@
+.. _golang-enterprise-authentication-mechanisms:
+
 ====================================
 Enterprise Authentication Mechanisms
 ====================================
 
-.. default-domain:: mongodb
-
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-In this guide, we show you how to authenticate with MongoDB using each
-**authentication mechanism** available exclusively in the MongoDB Enterprise
-Edition.
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available in the MongoDB Enterprise Edition.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
 
-You can use the following mechanisms with the latest version of the MongoDB
-Enterprise Edition:
+You can use the following authentication mechanisms with the latest version of
+the MongoDB Enterprise Edition.
 
-Kerberos (GSSAPI)
-LDAP (PLAIN)
+- :ref:`GSSAPI/Kerberos <golang-kerberos>`
+- :ref:`LDAP (Plain) <golang-LDAP>`
 
-To authenticate using another mechanism, see
-:ref:`golang-authentication-mechanisms`. To learn more about 
-establishing a connection to your MongoDB cluster, see 
+To authenticate using another mechanism, see the
+:ref:`<golang-authentication-mechanisms>` fundamentals page. For
+more information on establishing a connection to your MongoDB cluster, see the
 :ref:`golang-connection-guide`.
 
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using the following methods:
+
+- Using a **connection string**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- Specifying credentials and an authentication mechanism in the
+  ``Credential`` type from the {+driver-short+}. You can then use your
+  ``Credential`` struct to provide options for the ``Client``.
+
+This guide provides information on using types and methods provided by
+the {+driver-short+} for enterprise authentication, rather than using a
+connection string. For more information on using a connection string for
+enterprise authentication, see the :manual:`manual entry on Connection
+String URIs </reference/connection-string/#authentication-options>`.
+
+.. _golang-kerberos:
+
+Authenticate with GSSAPI/Kerberos
```

Comment:
Issue: I noticed three different prepositions used interchangeably for "authenticate" even though what it's modifying is one of two different things.

Suggestion: 
I think we should avoid "with" whenever possible since it is the least specific.

When describing the mechanism: "Authenticate using <auth mechanism>". 
When describing the server/srivce: "Authenticate to <server/service>".